ultra/ultra/scenarios/generate_scenarios.py

```diff
@@ -700,7 +701,11 @@ def build_scenarios(
     level_config = task_config["levels"][level_name]
     scenarios_dir = os.path.dirname(os.path.realpath(__file__))
     task_dir = f"{scenarios_dir}/{task}"
-    pool_dir = f"{scenarios_dir}/pool"
+
+    if pool_dir is None:
+        pool_path = os.path.join(scenarios_dir, "pool")
+    else:
+        pool_path = os.path.join(scenarios_dir, pool_dir)
```

Comment:
If the analysis scenarios are necessary to keep, I wonder if we can have an `pool/analysis_pool/` directory for the maps used for analysis, and then a default `pool/experiment_pool/`.